common/app/services/repositories.scala

```diff
@@ -18,7 +18,7 @@ trait Index extends ConciergeRepository with Collections {
 
   val contentApiClient: ContentApiClient
   val sectionsLookUp: SectionsLookUp
-  private val rssFields = s"${QueryDefaults.trailFields},byline,body,standfirst"
+  private val rssFields = s"${QueryDefaults.trailFields},byline,standfirst"
```

Comment:
The body field size is unbounded for live blogs. 
Not having to serve it for lists of liveblogs on liveblog RSS tag feeds removes some of the outlier large requests which CAPI has to marshall.